private/test/attachments/financial_year_exchanges/11/original.csv

```diff
@@ -0,0 +1,3 @@
+Jour,Numéro de compte,Journal,Tiers,Numéro de pièce,Libellé écriture,Débit,Crédit,Lettrage
```

Comment:
All these files in `private/test` must be removed. A `git checkout -f master -- private/test` should make the job